students/ramesh/ramesh.out

```diff
@@ -0,0 +1 @@
+new code
```

Comment:
what's this file? do you want it in git?

beware of `git add *` -- it can very easily add support files you don't want in the repo